changelog/REzNfQAnQ7mBG9FpYQV6Jw.md

```diff
@@ -0,0 +1,8 @@
+audience: worker-deployers
+level: patch
```

Comment:
```suggestion
level: minor
```

Could see an argument for a minor bump here since you're adding metrics/functionality. Won't block on it either way though.